src/App.tsx

```diff
@@ -15,36 +16,114 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Default = 'default',
+  Alphabetical = 'alphabetical',
+  Length = 'length',
+}
+
+interface TrackStatus {
+  reversed: boolean;
+  sortType: SortType;
+}
+
+const defaultState: TrackStatus = {
+  reversed: false,
+  sortType: SortType.Default,
+};
+
 export const App: React.FC = () => {
+  const [trackStatus, setTrackStatus] = useState<TrackStatus>(defaultState);
+
+  const { reversed, sortType } = trackStatus;
+
+  const getGoods = () => {
+    const goods = [...goodsFromServer];
+
+    if (sortType === SortType.Alphabetical) {
+      goods.sort((a, b) => a.localeCompare(b));
+    } else if (sortType === SortType.Length) {
+      goods.sort((a, b) => a.length - b.length);
+    }
```

Comment:
Please, prefer `switch/case` over multiple `if/else`

It may not be critical on two possible SortTypes, but still readability of the code suffers.